### ISSUE TITLE
error thrown when initialValues is missing array field

### ISSUE THREAD

**Issue by rhythnic** [erikras/redux-form#482]
By the way, thanks for your effort in making and maintaining this great library.

When the fields array contains an array field
```
const fields = {
  'title',
  'tags[]'
}
```
and the initialValues object is missing the array property
```
const initialValues = {
  title: 'My Title'
}
```
there is an error thrown from within getValues.js, line 27
```
TypeError: array.map is not a function
```

**Comment by erikras** [comment]
Fix published in [`v4.0.7`](https://github.com/erikras/redux-form/releases/tag/v4.0.7).

**Comment by trashgenerator** [comment]
Seems now it's opposite bug. If you specify 
```
const initialValues = {
  title: 'My Title',
  tags: [],
}
```
you will get 
```
Uncaught TypeError: array.forEach is not a function(anonymous function) 
@ getValues.js:24
```
